### ISSUE TITLE
Source code mode does not work: `TypeError: srclink is null`

### ISSUE THREAD

**Issue by TethysSvensson** [huhu/rust-search-extension#282]
When I try to use the example `src:try_fold`, it takes me to the page https://doc.rust-lang.org/std/convert/trait.TryInto.html?mode=src#tymethod.try_into but does not redirect to the source code page.

In my console I get this error:

```
TypeError: srclink is null   rust-src-navigate.js:16:13
    <anonymous>              moz-extension://18991aa0-92cb-467a-87ba-47b9e7adfa87/script/rust-src-navigate.js:16
    sendSyncMessage          moz-extension://ea808e03-b351-479e-8852-01e93fe7a1db/nscl/common/SyncMessage.js:244
    refetch                  moz-extension://ea808e03-b351-479e-8852-01e93fe7a1db/content/staticNS.js:124
    fetchLikeNoTomorrow      moz-extension://ea808e03-b351-479e-8852-01e93fe7a1db/content/staticNS.js:139
    fetchPolicy              moz-extension://ea808e03-b351-479e-8852-01e93fe7a1db/content/staticNS.js:94
    fetchPolicy              moz-extension://ea808e03-b351-479e-8852-01e93fe7a1db/content/staticNS.js:84
```

My system:

* **Browser**: Firefox version 126.0.1
* **Extension**: version 1.13.0, last updated at May 29, 2024
* **System**: NixOS 24.04

**Comment by hisriver** [comment]
Hi, @TethysSvensson. Thanks for the feedback, this has been fixed via https://github.com/huhu/rust-search-extension/commit/56039cd56a24dd23e20e090eb5f821616d6b34d2

**Comment by Folyd** [comment]
Hi, @TethysSvensson. Thanks for the feedback, this has been fixed via https://github.com/huhu/rust-search-extension/commit/56039cd56a24dd23e20e090eb5f821616d6b34d2

**Comment by Folyd** [comment]
Rust Search Extension v2.0 has been released. It has been fixed in v2.0.
https://github.com/huhu/rust-search-extension/releases/tag/v2.0.0
